### ISSUE TITLE
Questions about the experiments shown in Table 4

### ISSUE THREAD

**Issue by jszgz** [kenshohara/3D-ResNets-PyTorch#182]
“Top-1 accuracies on UCF-101 and HMDB-51.  Allaccuracies are averaged over three splits.” Did you train on three splits or only train on one split but test on three split? And is it fair to use ReduceLRonPlateau based on losses calculated from testlist01.txt(valition subset in ucf101_1.json) and then test acc on testlist01.txt?

**Comment by cotton-ahn** [comment]
I am also concerned about this issue. Is it fair to use ReduceLRonPlateau based on the validation loss? actually the "validation" in HMDB51 in here stands for the test dataset.
